controllers/providers/dns/infoblox.go

```diff
@@ -227,6 +227,7 @@ func (p *InfobloxProvider) saveHeartbeatTXTRecord(objMgr *ibcl.ObjectManager, gs
 		log.Info().
 			Str("HeartbeatTXTName", heartbeatTXTName).
 			Msg("Creating split brain TXT record")
+		//nolint:gosec
```

Comment:
@kuritka, can we document the skipping of this linter? Without additional comment, it looks slightly insecure :D

Reply:
Hi @ytsarev , sure, amended